### ISSUE TITLE
RSS main link destination

### ISSUE THREAD

**Issue by nclm** [ArtskydJ/comicsrss.com#142]
Hi, thanks for making this, that’s how I follow Nancy :)

I somehow often find myself clicking on the feed name in my reader, expecting it to lead me to the comic page. However, it goes to a comicsrss page instead with some message about a preview feature being removed.

Would it be possible to direct to the actual comic page instead? I feel that would be more useful.

Basically replacing:
```
<link>https://www.comicsrss.com/preview/nancy</link>
```

by
```
<link>https://www.gocomics.com/nancy</link>
```

in the RSS header.

**Comment by ArtskydJ** [comment]
Ah, that would be more useful.  Fixed in ea18a869982ef0aa562dac3a2468f7f6c36e2351 and 05c4d77b859285fa1d2307c6b36711b088c21a4f.

**Comment by nclm** [comment]
Thanks!!
